library/src/skins/default/src/sass/modules/tool/sitestats/_sitestats.scss

```diff
@@ -134,12 +134,17 @@
   /** STAT WIDGET: Mini panels */
   .widget .ministatContainer {
     margin-left: 100px;
+    border-left: 1px solid var(--sakai-border-color);
   }
 
   .widget .ministat {
     padding: 0 10px;
     border-left: 1px solid var(--sakai-border-color);
     text-align: center;
+	@media #{$phone}{
+	padding: 0 60px;
```

Comment:
Spacing

Reply:
@Sharadhi98 If you could just fix the indentation on these lines, we can get this merged. We try to use spaces in our files where we can, and this file seems to be using 2 spaces.

Reply:
Thank you, I pushed the updated code.